src/pages/ReimbursementAccount/BankInfo/substeps/Confirmation.tsx

```diff
@@ -92,19 +86,21 @@ function Confirmation({reimbursementAccount, reimbursementAccountDraft, onNext})
                         </View>
                     )}
                     {setupType === CONST.BANK_ACCOUNT.SUBSTEP.PLAID && (
+                        // @ts-expect-error TODO: remove this once MenuItem (https://github.com/Expensify/App/issues/25144) is migrated to TS
```

Comment:
`MenuItem` should already be migrated

Reply:
Must've happend recently. Our feature branch got updated yesterday and when I started this migration it wasnt in ts yet. Will remove the expect-error comments